### ISSUE TITLE
[$500] Inviting someone first time mentioning email does not receive message 

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#30896]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.3.95.5
**Reproducible in staging?:** needs reproduction
**Reproducible in production?:** needs reproduction
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @quinthar 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1699163020890799

## Action Performed:
1. Open NewDot
2. Create a room
3. Invite new user by mentioning the email

## Expected Result:
User should receive email message to the invited user

## Actual Result:
No email received

## Workaround:
unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~0135f955b9d989a1e1</li>
        <li>Upwork Job ID: 1722007006840905728</li>
        <li>Last Price Increase: 2023-11-07</li>
    </ul>
</details>

**Comment by puneetlath** [comment]
@johncschuster this will need to be internal. cc @jasperhuangg

**Comment by jasperhuangg** [comment]
I think this is a dupe of something that @marcochavezf is already working on

https://github.com/Expensify/App/issues/30919

**Comment by johncschuster** [comment]
Sweet! Thanks for the heads up, @jasperhuangg! I'll put this on hold while we wait on https://github.com/Expensify/App/issues/30919 to be deployed.

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold

**Comment by johncschuster** [comment]
Still on hold. Downgrading to `Weekly` while we wait.

**Comment by johncschuster** [comment]
I will be OOO starting Monday, December 18, and will return Tuesday, January 2.

**Current status:** 
This is on hold for https://github.com/Expensify/App/issues/30919
No proposals

If this issue is open when I'm back from OOO, I'll take it back over. Thank you!

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
Hold

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
Hold.

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
On hold.

**Comment by lschurr** [comment]
On hold.

**Comment by marcochavezf** [comment]
https://github.com/Expensify/App/issues/30919 was completed, applying the retest-weekly label

**Comment by lschurr** [comment]
Hey @marcochavezf - should this be tied to vip-vsb?

**Comment by marcochavezf** [comment]
It should be fixed already, the linked issue is already closed

**Comment by lschurr** [comment]
Great. Closing.
